plenum/test/tracker/test_ledger_uncommitted_tracker.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import random
+
+from plenum.common.util import randomString
+from plenum.common.ledger_uncommitted_tracker import LedgerUncommittedTracker
+from common.exceptions import PlenumValueError, LogicError
+
+
+@pytest.fixture()
+def state_root():
+    return randomString(32).encode()
+
+
+@pytest.fixture()
+def make_tracker():
+    return LedgerUncommittedTracker()
+
+
+def test_error_with_no_state_root_track_uncommitted(make_tracker):
+    with pytest.raises(PlenumValueError):
+        make_tracker.apply_batch("", 12)
+
+
+def test_error_with_invalid_ledger_size(make_tracker, state_root):
+    with pytest.raises(PlenumValueError):
+        make_tracker.apply_batch(state_root, random.randint(-99, 0))
+
+
+def test_error_with_revert_empty_tracker(make_tracker):
+    with pytest.raises(LogicError):
+        make_tracker.reject_batch()
+
+
+def test_build_uncommitted_and_reject_batch(make_tracker):
```

Comment:
Can you please add more tests for each API method?
Also it makes sense to test that calling `apply_batch` and/or `revert_batch` works as expected.